docs/intl.md

```diff
@@ -0,0 +1,155 @@
+  o To test the new Spark component regional date formatting in your application, set the global variable on the window object:
+
+      ```
+      window['GUX_OPTIONS_enableRegionalDates']= true
+      ```
+
+## Feature Toggling
+
+- All regional date functionality should be feature toggled. We want to avoid confusing customers with different date time behavior across applications
+- The Spark component functionality is internally feature toggled and when it is time to GA this feature, will be globally enabled via a cookie set in web directory
+- If your app resides outside of web directory, this cookie should still take effect after the user navigates to web directory the first time
+- All regional date functionality that is outside of the Spark components should be feature toggled behind the feature toggle for the epic, `PURE-2946_InternationalizationOfDatesAndTimes`
+
+## Using components for new date/time regional features
+
+All Spark components that relate to date or time leverage support the new regional date/time features. Here is a list of all the components that are affected by these changes:
+
+- gux-calendar
+- gux-date
+- gux-datepicker
+- gux-date-time
+- gux-month-picker
+- gux-month-calendar
+- gux-time
+- gux-time-picker
+
+If you already use these components in your application and are using the latest versions of either V3 or V4, please verify that the new updates are working as expected in your application. As mentioned above, verifying these changes can be done by settting the global variable on the window object: `window['GUX_OPTIONS_enableRegionalDates']= true`
+If you see any issues when verifying these new features, please create a COMUI ticket or reach out in the Common UI Development room.
+
+## New Spark Utilities:
+
+In addition to updating the Spark components, we have also added new utilities to the `genesys-spark` package to assist with date and time formatting. These utilities standardize date and time formatting practices in Spark components and can serve as utility functions for applications. The utilities are proxies to the browser’s `Intl` object, and will determine the user’s region from the browser configuration of the current language setting if a region is not specified.
+
+Importing the Intl helpers from `genesys-spark` is only possible in `v4`, due to limitations in how Stencil builds packages in `v3`, which makes export of JS APIs not possible. If the Intl helpers are needed in your application but you are still using v3 of genesys-spark-components, it is recommend to import v4 of `genesys-spark` in addition. Using `v4` of genesys-spark alongside `v3` of genesys-spark-components will be reliable as long as registerSparkComponents is not called, and since the genesys-spark package does not contain any component code, using both should not affect the bundle size in your applications.
```

Comment:
```suggestion
Importing the Intl helpers from `genesys-spark` is only possible in `v4`, due to limitations in how Stencil builds packages in `v3`, which makes export of JS APIs not possible. If the Intl helpers are needed in your application but you are still using `v3` of genesys-spark-components, it is recommend to import `v4` of `genesys-spark` in addition. Using `v4` of genesys-spark alongside `v3` of genesys-spark-components will be reliable as long as registerSparkComponents is not called, and since the genesys-spark package does not contain any component code, using both should not affect the bundle size in your applications.
```